packages/table/preview/index.tsx

```diff
@@ -468,6 +528,31 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
     // State updates
     // =============
 
+    // designed to be called from componentWillMount and componentWillUpdate, hence it expects nextProps
+    private syncCellContent = (nextState?: IMutableTableState) => {
```

Comment:
you could pull that `isNextStateDefined` check up a level to `componentWillUpdate` itself, cuz it only applies to that one usage of this function.